conanfile.py

```diff
@@ -9,25 +9,8 @@ class TestConan(ConanFile):
     url = "None"
     license = "None"
 
-    def build(self):
-        if not os.path.exists("build/include"):
-            os.makedirs("build/include")
-            
-        if not os.path.exists("build/lib"):
-            os.makedirs("build/lib")
-
-        header_files = glob.glob("code/**/*.h", recursive=True)
-        for file in header_files:
-            if os.path.isfile(file):
-                shutil.copy2(file, "build/include/" + os.path.basename(file))
-                
-        lib_files = glob.glob("obj/**/*.a", recursive=True)
-        for file in lib_files:
-            if os.path.isfile(file):
-                shutil.copy2(file, "build/lib/" + os.path.basename(file))
-
     def package(self):
-        self.copy("build")
+        self.copy("*", dst="lib", src="obj/libs")
```

Comment:
Yep I see what you mean.

The convention we have used in our project passes all the include paths to the compiler so there are no nested #includes.